operator/v1/types_console.go

```diff
@@ -21,13 +21,27 @@ type ConsoleSpec struct {
 	// customization is used to optionally provide a small set of
 	// customization options to the web console.
 	// +optional
-	Customization ConsoleCustomization `json:"customization"`
+	Customization ConsoleCustomization `json:"customization,omitempty"`
+	// providers contains configuration for using specific service providers.
+	Providers ConsoleProviders `json:"providers,omitempty"`
 }
 
 type ConsoleStatus struct {
 	OperatorStatus `json:",inline"`
 }
 
+type ConsoleProviders struct {
+	// statuspage contains ID for statuspage.io page that provides status info about.
+	// +optional
+	Statuspage *StatuspageProvider `json:"statuspage,omitempty"`
```

Comment:
Here `omitempty` does make a difference and it shows good usage.

Reply:
`StatusPage`

Reply:
> `StatusPage`

The name of the service is `Statuspage`, one word with a lowercase `p`.

https://www.statuspage.io/

Reply:
OK, I wasn't aware of that, it can stay that, in that case.